apps/api/drizzle/0045_broad_sentry.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE "competition_prize_pools" (
```

Comment:
new table stores the two prize pools in WEI. we still don't have a way to update these value. It will be done in future PR